bundle/manifests/nic-configuration-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,373 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "configuration.net.nvidia.com/v1alpha1",
+          "kind": "NicConfigurationTemplate",
+          "metadata": {
+            "name": "connectx6-config",
+            "namespace": "nic-configuration-operator"
+          },
+          "spec": {
+            "nicSelector": {
+              "nicType": "101b",
+              "pciAddresses": [
+                "0000:03:00.0",
+                "“0000:04:00.0”"
+              ],
+              "serialNumbers": [
+                "MT2116X09299"
+              ]
+            },
+            "nodeSelector": {
+              "feature.node.kubernetes.io/network-sriov.capable": "true"
```

Comment:
IMO, it's better to add selector by vendor name

Reply:
done